lib/src/command/lish.dart

```diff
@@ -77,7 +95,7 @@ class LishCommand extends PubCommand {
         return log.progress('Uploading', () async {
           // TODO(nweiz): Cloud Storage can provide an XML-formatted error. We
```

Comment:
This comment looks like it applies to the POST request, perhaps it should be moved there or deleted.